arches_controlled_lists/src/arches_controlled_lists/widgets/ReferenceSelectWidget/components/ReferenceSelectWidgetEditor.vue

```diff
@@ -65,15 +116,28 @@ async function getOptions() {
 
 // let timeout: ReturnType<typeof setTimeout>;
 
-function resolver(e: FormFieldResolverOptions) {
-    validate(e);
+function resolver(options: FormFieldResolverOptions) {
     // return new Promise((resolve) => {
     //     if (timeout) clearTimeout(timeout);
 
     //     timeout = setTimeout(() => {
     //         resolve(validate(e));
     //     }, 500);
     // });
+    const nodeAlias = props.nodeAlias;
+    if (options.value) {
+        options.value = Object.entries(options.value).reduce<string[]>(
+            (keys, [key, val]) => {
+                if (val === true) keys.push(key);
+                return keys;
+            },
+            [],
+        );
+    }
+    validate(options);
+    return {
+        values: { [nodeAlias]: options.value }, //, errors
```

Comment:
what's with `//, errors`?

Reply:
Hint for the next implementer to remember to include errors with the return.

Reply:
Remove?

Reply:
if you're already in it, yeah. Also the other commented code has been removed from the other widgets, if you'd like to follow suit here. Either way on this one, I don't feel strongly.